cfme/configure/tasks.py

```diff
@@ -160,3 +160,47 @@ def all_other_tasks(**filter_kwargs):
     Returns: List of dicts.
     """
     return _get_tasks("tasks_all_other", **filter_kwargs)
+
```

Comment:
Forgive me here, but the is_analysis_finished seems generically like a is_task_finished? Wouldn't it make more sense to put this as a generic method for a task? Then the ```is_{specific}_finished``` would become ```is_analysis_finished``` methods on the relevant objects.
@mfalesni comments?